projects/level-money/index.js

```diff
@@ -43,10 +35,12 @@ const TOKEN_CONTRACTS = [
 
 module.exports = {
     ethereum: {
-        tvl: sumTokensExport({ 
-          owners: [LEVEL_STAKING_CONTRACT, ...LEVEL_RESERVE_MANAGERS, ...LEVEL_WRAPPED_TOKENS], 
-          tokens: TOKEN_CONTRACTS,
-        }),
+        tvl: async (api) => {
+            const supply = await api.call({ abi: 'erc20:totalSupply', target: LVL_USD })
+            api.add(LVL_USD, supply)
```

Comment:
please count the collateral instead of totalSupply

Reply:
hey @waynebruce0x, the collateral backs the total supply 1:1. noticed some other stablecoins doing this as well (ie USDe). do you still need us to count the collateral?

Reply:
yeah please